### PR TITLE
Replace `bou.ke/monkey`

### PR DESCRIPTION
**Pull request by Eun**
Due to License problems, we have to replace `bou.ke/monkey`

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,12 +1,22 @@
-bou.ke/monkey v1.0.2 h1:kWcnsrCNUatbxncxR/ThdYqbytgOIArtYWqcQLQzKLI=
-bou.ke/monkey v1.0.2/go.mod h1:OqickVX3tNx6t33n1xvtTtu85YN5s6cKwVug+oHMaIA=
+github.com/agiledragon/gomonkey/v2 v2.1.0 h1:+5Dbq8a1fn89IgVk35O233R41FH0nBKFPn50wDZpNs0=
+github.com/agiledragon/gomonkey/v2 v2.1.0/go.mod h1:ap1AmDzcVOAz1YpeJ3TCzIgstoaWLA6jbbgxfB4w2iY=
 github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1/go.mod h1:wJfORRmW1u3UXTncJ5qlYoELFm8eSnnEO6hX4iZ3EWY=
```
Comment (wlMalk) [inline go.sum:5]: Is there any reason `gopherjs` shows up here?

Comment (Eun) [reply go.sum:5]: Looks like a dead import from `gomonkey`

```
╰$ grep -R gopherjs .
./go.sum:github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1/go.mod h1:wJfORRmW1u3UXTncJ5qlYoELFm8eSnnEO6hX4iZ3EWY=
./vendor/github.com/agiledragon/gomonkey/v2/go.sum:github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1 h1:EGx4pi6eqNxGaHF6qqu48+N2wcFQ5qg5FXgOdqsJ5d8=
./vendor/github.com/agiledragon/gomonkey/v2/go.sum:github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1/go.mod h1:wJfORRmW1u3UXTncJ5qlYoELFm8eSnnEO6hX4iZ3EWY=
```

Comment (altJake) [reply go.sum:5]: Is it just a matter of running go tidy then? 🤔

**Comment by altJake** [comment]
@Eun 
Just one comment/question from my side: how will we "release it"?
Do we need to keep any sort of monitoring in our heads? Or it is a simple upgrade of the package version in the `talon-service` repo?

File: go.sum (hunk 1 above)
Comment (Eun) [reply go.sum:5]: `go tidy` should be it, but changes nothing in the repo. Anyways its not hurting, since `gopherjs` does not appear in the `vendors`

**Comment by Eun** [comment]
@altJake Simple module update would be enough.

**Review by altJake — APPROVED** [review]

**Pull request merged**